src/DynamoCoreWpf/Views/Core/DynamoView.xaml.cs

```diff
@@ -702,6 +703,7 @@ void DynamoViewModelRequestShowPackageManagerSearch(object s, EventArgs e)
             if (!DisplayTermsOfUseForAcceptance())
                 return; // Terms of use not accepted.
 
+            var cmd = Analytics.TrackCommandEvent("SearchPackage");
```

Comment:
Dispose is mainly to track the time duration spent on the dialog, hence dispose is called when the window closes. It won't leak any resources.